projboard/tests/conftest.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from projboard.models.article import Article, User, View_Article, Like, Subject
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+@pytest.fixture
+def create_subjects():
+    sup = Subject(name='N')
+    sup1 = Subject(name='R')
+    sup.save()
+    sup1.save()
+    return [sup, sup1]
+
+
+@pytest.fixture
+def create_user():
```

Comment:
This fixture returns a list of users, so `create_users` is more appropriate.  
And to follow the first comment, `users` would be a better choice

Reply:
Done

Reply:
I think we first named it right like user and subject but when we did the presentation we got notes from monetors that we should change it to create_somthing